nat-lab/tests/utils/bindings.py

```diff
@@ -0,0 +1,49 @@
+from typing import Callable, Optional
+
+from uniffi.telio_bindings import *  # pylint: disable=wildcard-import, unused-wildcard-import  # isort: skip
```

Comment:
The idea here is that this file exports all the types from the generated bindings so that anywhere you want to use those types you can import them from this file (cleans up the imports a bit since there are utilities here and then you don't have separate imports for "direct from generated bindings" and "from binding utilities"). It would definitely work without the wildcard but then if you want to use a type that wasn't explicitly exported already you have to add it here manually, to what would become a very big list of imports. I generally don't like wildcard imports but I think in this case it's the better choice